src/main/java/dansplugins/factionsystem/managers/ConfigManager.java

```diff
@@ -120,6 +120,9 @@ public void handleVersionMismatch() {
         if (!MedievalFactions.getInstance().getConfig().isBoolean("territoryAlertPopUp")) {
             MedievalFactions.getInstance().getConfig().addDefault("territoryAlertPopUp", true);
         }
+        if (!MedievalFactions.getInstance().getConfig().isBoolean("territoryIndicatorActionbar")) {
+            MedievalFactions.getInstance().getConfig().addDefault("territoryIndicatorActionbar", true);
```

Comment:
The "territoryIndicatorActionbar" config option will have to be added to the following methods:
- setConfigOption()
- saveConfigDefaults()
- sendPageTwoOfConfigList()

Reply:
I made the corresponding changes but I don't know how to add them to this Pull Request.
Should I close this one and open a new one including the changes?

Reply:
@dmccoystephenson 
Ok, it's in 31c8056 as part of this PR. So this change request is probably resolved.

Reply:
Looks good, thank you for making those changes!